### ISSUE TITLE
opea/llm-faqgen-tgi failed to launch

### ISSUE THREAD

**Issue by lianhao** [opea-project/GenAIComps#700]
Starting from today, llm-faqgen-tgi failed to launch with the following logs:

```
Installing collected packages: pydantic-core, pydantic, langsmith, langchain-core, langserve
  Attempting uninstall: langsmith
    Found existing installation: langsmith 0.1.116
    Uninstalling langsmith-0.1.116:
      Successfully uninstalled langsmith-0.1.116
  WARNING: The script langsmith is installed in '/home/user/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
  Attempting uninstall: langchain-core
    Found existing installation: langchain-core 0.1.52
    Uninstalling langchain-core-0.1.52:
      Successfully uninstalled langchain-core-0.1.52
Successfully installed langchain-core-0.3.1 langserve-0.3.0 langsmith-0.1.121 pydantic-2.9.2 pydantic-core-2.23.4
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
langchain-community 0.0.38 requires langchain-core<0.2.0,>=0.1.52, but you have langchain-core 0.3.1 which is incompatible.
langchain-text-splitters 0.0.2 requires langchain-core<0.3,>=0.1.28, but you have langchain-core 0.3.1 which is incompatible.
langchain 0.1.16 requires langchain-core<0.2.0,>=0.1.42, but you have langchain-core 0.3.1 which is incompatible.
/home/user/.local/lib/python3.11/site-packages/langchain/chains/combine_documents/__init__.py:3: LangChainDeprecationWarning: As of langchain-core 0.3.0, LangChain uses pydantic v2 internally. The langchain_core.pydantic_v1 module was a compatibility shim for pydantic v1, and should no longer be used. Please update the code to import from Pydantic directly.

For example, replace imports like: `from langchain_core.pydantic_v1 import BaseModel`
with: `from pydantic import BaseModel`
or the v1 compatibility namespace if you are working in a code base that has not been fully upgraded to pydantic 2 yet.         from pydantic.v1 import BaseModel

  from langchain.chains.combine_documents.reduce import (
Traceback (most recent call last):
  File "/home/user/comps/llms/faq-generation/tgi/llm.py", line 7, in <module>
    from langchain.chains.summarize import load_summarize_chain
  File "/home/user/.local/lib/python3.11/site-packages/langchain/chains/summarize/__init__.py", line 8, in <module>
    from langchain.chains.combine_documents.base import BaseCombineDocumentsChain
  File "/home/user/.local/lib/python3.11/site-packages/langchain/chains/combine_documents/__init__.py", line 3, in <module>
    from langchain.chains.combine_documents.reduce import (
  File "/home/user/.local/lib/python3.11/site-packages/langchain/chains/combine_documents/reduce.py", line 124, in <module>
    class ReduceDocumentsChain(BaseCombineDocumentsChain):
  File "/home/user/.local/lib/python3.11/site-packages/pydantic/_internal/_model_construction.py", line 224, in __new__
    complete_model_class(
  File "/home/user/.local/lib/python3.11/site-packages/pydantic/_internal/_model_construction.py", line 587, in complete_model_class
    schema = gen_schema.clean_schema(schema)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/user/.local/lib/python3.11/site-packages/pydantic/_internal/_generate_schema.py", line 595, in clean_schema
    schema = validate_core_schema(schema)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/user/.local/lib/python3.11/site-packages/pydantic/_internal/_core_utils.py", line 570, in validate_core_schema
    return _validate_core_schema(schema)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
pydantic_core._pydantic_core.SchemaError: Invalid Schema:
definitions.schema.model.config.extra_fields_behavior
  Input should be 'allow', 'forbid' or 'ignore' [type=literal_error, input_value=<Extra.forbid: 'forbid'>, input_type=Extra]
    For further information visit https://errors.pydantic.dev/2.9/v/literal_error
```

**Comment by lianhao** [comment]
comment to take this bug

**Comment by lianhao** [comment]
This is due to the latest released langserve v0.3.0, langserve v0.2.3 works fine.

**Comment by lianhao** [comment]
closed by PR #704
